pilot/pkg/networking/util/util.go

```diff
@@ -257,15 +257,21 @@ func SortVirtualHosts(hosts []route.VirtualHost) {
 	})
 }
 
-// IsProxyVersionGE11 checks whether the given Proxy version is greater than or equals 1.1.
-func IsProxyVersionGE11(node *model.Proxy) bool {
-	ver, _ := node.GetProxyVersion()
-	return ver >= "1.1"
+// IsIstioVersionGE12 checks whether the given Istio version is greater than or equals 1.2.
+func IsIstioVersionGE12(node *model.Proxy) bool {
```

Comment:
When this function used? As we donot need check version.

Same with `IsIstioVersionGE12`.

Reply:
not right now. But #15802 will be using it (the 1.3 thing).